src/screens/LinkContextScreen/elements/Header.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-expressions */
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `/* eslint-disable no-unused-expressions */` at the top of the file suggests that there might be some expressions in this file that are not used. It's generally a good practice to remove unused code to improve readability and maintainability. If the expressions are indeed needed, consider adding a comment explaining why they're necessary.

```diff
- /* eslint-disable no-unused-expressions */
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->